shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -67,10 +67,15 @@ void PostAccessibilityNotification(UIAccessibilityNotifications notification,
   return [[platform_view_->GetOwnerViewController().get().engine textInputPlugin] textInputView];
 }
 
-void AccessibilityBridge::AccessibilityFocusDidChange(int32_t id) {
+void AccessibilityBridge::AccessibilityObjectDidBecomeFocused(int32_t id) {
   last_focused_semantics_object_id_ = id;
 }
 
+void AccessibilityBridge::AccessibilityObjectDidLoseFocus(int32_t id) {
+  if (last_focused_semantics_object_id_ == id)
+    last_focused_semantics_object_id_ = -1;
```

Comment:
Can you do a define for this magic value, something like `kSemanticObjectIdInvalid`?